pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/mercury/MercurySampleDao.java

```diff
@@ -40,7 +41,7 @@ public class MercurySampleDao implements Dao<MercurySampleDto> {
                     + "From ddp_mercury_sequencing where ddp_participant_id = ? group by (tissue_id) ) as table2 "
                     + "on table2.seqt = table1.tissue_id "
                     + "LEFT JOIN (select order_id, mercury_sequencing_id, order_status, order_date, mercury_pdo_id "
-                    + "FROM ddp_mercury_sequencing WHERE ddp_participant_id = ?) as table3 ON(table3.mercury_sequencing_id = table2.ms_id)";
+                    + "FROM ddp_mercury_sequencing WHERE ddp_participant_id = ?) as table3 ON(table3.mercury_sequencing_id = table2.ms_id) ";
```

Comment:
Is this change intentional?

Reply:
Yeah I like to have an extra space at the end in case we need to concat the query with some other condition